test/browser/run-test.sh

```diff
@@ -25,10 +25,17 @@ export TEST_AUDIT_NO_SELINUX=1
 
 EXCLUDES=""
 
+if [ "$ID" = "rhel" ]; then
```

Comment:
Actually, we should not exclude these tests for RHEL 8, as they already work there. Can you please add `-a "${VERSION_ID#9}" != "$VERSION_ID` to only exclude them for RHEL 9?

Reply:
I just checked that we don't call *this* script in downstream RHEL 8 testing, so ok for now.